winui/AutoComplete/Leading-and-Trailing-view.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Leading and Trailing View with WinUI AutoComplete | Syncfusion
+description: Learn here all about Leading and Trailing View support in Syncfusion WinUI AutoComplete control and more.

```

Comment:
Learn how to add leading and trailing view in Syncfusion WinUI AutoComplete control and more.